LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 The Ethereum Foundation
+Copyright (c) 2019 The <PROJECT_NAME> contributors
```

Comment:
According to https://haacked.com/archive/2006/01/26/WhoOwnstheCopyrightforAnOpenSourceProject.aspx/ we should potentially keep "The Ethereum Foundation" as the copyright owner as it doesn't seem to actually benefit the project or the contributors to attribute it to the contributors.

- It makes the project no more "open source" or permissive
- It makes it harder to try and enforce the copyright since the copyright is held by some nebulous collective entity.

Reply:
Yeah, this argument makes a lot of sense to me. For example, if we wanted to relicense under Apache instead of MIT, it would require a lot less work.

*But*... in order for this to work, we have to get copyright assignment from all contributors, which we're not currently doing AFAIK. So we should probably set that up. There was some recent relevant discussion about doing that in AllCoreDevs. Roughly, I think the solution is to have a bot that tracks which contributors have signed over their copyright, and blocks merge until that happens. Then we have to go back to all the old contributors to get them to assign their copyright from previous contributions. (This *may* not be necessary for bountied PRs, since it might fall under "work for hire")

Reply:
Alright, if that has legal implications, the let's keep it as is.